src/@next/components/organisms/CheckoutPayment/styles.ts

```diff
@@ -1,6 +1,8 @@
 import { media, styled } from "@styles";
 
-export const Wrapper = styled.div``;
+export const Wrapper = styled.div`
+  width: 1242px;
```

Comment:
why not width 100% or another solution to handle width of wrapper, not should hardcode.